rpc/doc.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package juju is devops distilled.
+//
+// Project homepage: https://github.com/juju/juju
+//
+// For more information please refer to the README.md file
+// in this directory.
+
```

Comment:
All of the above needs to go -- it really does not belong here.

Reply:
As an intro you can maybe use the following (check that it's correct):

> Package rpc provides the underpinnings of requests and responses (server.Request) and dispatching them to the API. 

The package defines a notion of request that has a named type, a version, and a method to be called, and provides a mechanism – the Find method – for matching a request to a callable function.

Reply:
Sure thanks teodora!